src/common/components/map/d3Components/shadowDefinitions.ts

```diff
@@ -0,0 +1,37 @@
+import { D3Selection } from '../../../types';
+import { SVG_SHADOW_ID } from './constants';
+
+interface Props {
+  svg: D3Selection;
+}
+
+export const shadowDefinitions = (props: Props) => {
+  render(props);
+};
+
+const render = ({ svg }: Props) => {
+  const defs = svg.append('defs');
+
+  const filter = defs
+    .append('filter')
+    .attr('id', SVG_SHADOW_ID)
+    .attr('filterUnits', 'userSpaceOnUse');
+
+  filter.append('feGaussianBlur')
```

Comment:
Code like this it's a bit cryptic for a noob on d3js, and there is no better way to write it, I think it would be a good idea to add a commented on each filter and comment what it does (IMHO in this case comments make sense).